### PR TITLE
Added playlist page

### PR DESCRIPTION
**Pull request by AbigailSalazar**
Hi, I added the route for playist page, which has two new components: 

**PlaylistHeader**

imagen

**SideInfo**

imagen

Together looks like this:
imagen

I want to work in the playlist list as well, let me know if that's okey and if the code it's good as well :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "@tauri-apps/api": "^1",
+    "juxtify": "file:",
```
Comment (BostonRohan) [inline package.json:18]: what does this do? we don't have a `juxtify` package or dep that we're referencing anywhere right?

Comment (AbigailSalazar) [reply package.json:18]: No. I didn't add it. I though It deleted when I got the new changes from the sidebar but it's there again.

Comment (AbigailSalazar) [reply package.json:18]: I don't know what it does. I tried to delete it but it automatically adds.

Comment (BostonRohan) [reply package.json:18]: weird. maybe some config in your local environment? I'll merge it in anyway it's not a big deal.

**Pull request merged**